carRental/src/main/java/com/amigoscode/chohort2/carRental/driverLicense/DriverLicenseDTO.java

```diff
@@ -0,0 +1,30 @@
+package com.amigoscode.chohort2.carRental.driverLicense;
+
+import com.amigoscode.chohort2.carRental.user.UserDTO;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+
+import java.time.LocalDate;
+
+@Setter
+@Getter
+@Accessors(chain = true)
+public class DriverLicenseDTO {
+
+    private Long id;
+
+    private Long userId;
+
+    private UserDTO userDTO;
+
```

Comment:
_Due to use of MapStruct_ mapping fields in dtos should have the same naming as in entities:
private UserDTO **user**;

Reply:
hummm can you change map struct to handle naming